app/controllers/grades_controller.rb

```diff
@@ -188,34 +188,34 @@ def mark_grading_allocation_completed
     # there should be at most one grading allocation per user per assignment,
     # and at most one grading allocation should not be abandoned at any given time
     # but prefer to find the one for the current user, first
-    alloc = (allocs.find{|a| a.who_grades_id == current_user.id} || allocs.find{|a| !a.abandoned && a.grading_completed.nil?})
```

Comment:
Everything from here through line 248 is indented incorrectly, and in any case is irrelevant to this PR.  Please revert.